### PR TITLE
St 86 tournament results

### PR DESCRIPTION
**Pull request by elexunix**
Hello, accept this!

### CODE REVIEW

File: app/templates/temp.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <tr>
+            <td>{{ i[0] }}</td>
+            <td>{{ i[1] }}</td>
+            <td>{{ i[2] }}</td>
+        </tr>
+    {% endfor %}
+    </table>
+</div>
+<hr>
+{% endblock %}
```
Comment (alexdany657) [inline app/templates/temp.html]: No new line at EOF

File: app/routes/all.py [hunk 2]
```diff
@@ -28,11 +29,20 @@ def problemset_id(strId):
...
 def showSource(subId):
-    title = "Code #" + subId
-    return render_template('source.html.j2', id = subId, code = useCasesAPI.getSubmissionCode(subId), info = info())
+    submission = storage.getSubmission(subId);
+    Info = info();
+    title = "Code #" + subId;
+    print(Info);
+    if Info[0] and Info[2] == submission.userId:
+        return render_template('source.html.j2', id = subId, code = useCasesAPI.getSubmissionCode(subId), info = info());
+    return render_template('ban.html.j2', info = info());
```
Comment (alexdany657) [inline app/routes/all.py]: No signs ';' at EOL

Comment (alexdany657) [inline app/routes/all.py]: Don't put ';' at EOL

Comment (alexdany657) [inline app/routes/all.py]: Don't put ';' at EOL

**Comment by elexunix** [comment]
hello again

**Pull request merged**